### PR TITLE
IPC Rework Pt1: Bugfixes

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
Atomizes #5442 

## Why It's Good For The Game
Bugs bad. IPCs good.

## Changelog
:cl:
code: Cleaned up IPC code in general.
feature: IPCs now show "dents" and "leaking" instead of "bruises" and "bleeding"
fix: VV health adjustment works on synthetics
fix: IPCs now actually have blood, oil, and it leaves decals when they are hurt
fix: IPCs are now actually immune to toxin damage.
fix: IPCs wont process reagents they weren't supposed to process.
refactor: Mild refactor to species metabolisms. 
/:cl:

### CODE REVIEW

File: code/modules/admin/view_variables/topic.dm [hunk 1]
```diff
@@ -89,10 +89,10 @@
 			var/newamt
 			switch(Text)
 				if("brute")
-					L.adjustBruteLoss(amount)
+					L.adjustBruteLoss(amount,,TRUE)
```
Comment (Archanial) [inline code/modules/admin/view_variables/topic.dm]: ?

Comment (Kapu1178) [reply code/modules/admin/view_variables/topic.dm]: sets second variable to null, and the third variable, FORCED, to true.

Comment (francinum) [reply code/modules/admin/view_variables/topic.dm]: Bad, set third using named argument.

Comment (Kapu1178) [reply code/modules/admin/view_variables/topic.dm]: fiiiiiiiiiiiiiine

**Review by francinum — APPROVED** [review]

**Pull request merged**